File: 2706-buy-two-chocolates/2706-buy-two-chocolates.cpp
class Solution {
public:
    int buyChoco(vector<int>& prices, int money) {
        int choc1=INT_MAX, choc2= INT_MAX;
        for(int i=0; i<prices.size(); i++){
            if(prices[i]<choc1){
                choc2= choc1;
                choc1= prices[i];
            }
            else if(prices[i]<choc2) choc2= prices[i];
        }
        if(choc1+ choc2> money) return money;
        return money- choc1- choc2;
    }
};

File: 2706-buy-two-chocolates/2706-buy-two-chocolates.java
class Solution {
    public int buyChoco(int[] prices, int money) {
        int mini1= Integer.MAX_VALUE, mini2= Integer.MAX_VALUE;
        for(int i=0; i<prices.length; i++){
            if(mini1> prices[i]){
                mini2= mini1;
                mini1= prices[i];
            }else if(mini2> prices[i]) mini2= prices[i];
        }
        return (mini1+mini2>money)? money: money- (mini1+mini2);
    }
}